packages/@lwc/engine-core/scripts/rollup.config.js

```diff
@@ -65,4 +65,6 @@ module.exports = {
             throw new Error(message);
         }
     },
+
+    external: ['@lwc/features', '@lwc/shared'],
```

Comment:
The list of external modules should be in sync with the `dependencies` in the `package.json` ([example](https://github.com/salesforce/lwc/pull/2025/files#diff-9bf184bd248f5ac6e8d18aa7104378ccd91574dcc044b3d784d9c306a84452baR41)).

Reply:
Done.